spec/services/message_employee_matcher_spec.rb

```diff
@@ -30,7 +30,7 @@
           Timecop.freeze(Time.parse("00:00:00 UTC")) do
             days_after_start = 3
             scheduled_message_time = Time.parse("12:00:00 UTC")
-            scheduled_message = create(:scheduled_message, days_after_start: days_after_start, time_of_day: scheduled_message_time)
+            scheduled_message = create(:scheduled_message, days_after_start: days_after_start + 1, time_of_day: scheduled_message_time)
```

Comment:
what does this change mean? I think it might no longer match the spec description with this change.

the test description is ""does not send messages to employees who are at a later time but on a different date" 

In this changed spec, do we still have a user who is at a later time but on a different date?

I can already feel my brain getting fried just thinking about this. This may be correct, just want to make sure I understand the implications.